README.md

```diff
@@ -124,39 +124,74 @@ a proxy tool such as [mitmproxy](https://mitmproxy.org), or even just
 a script.
 
 A particularly simple way to proxy requests to a private bitcoind node
-is to make use of the [`corsproxy`](https://www.npmjs.com/package/corsproxy)
-npm module. Instructions to install and run the module are on its
-[home page](https://www.npmjs.com/package/corsproxy).
+is to make use of [`nginx`](https://nginx.org/). Instructions to install
```

Comment:
I added that in the commit. In all honesty, CORS header support should be in the bitcoind RPC. By not having it there they are defaulting to a less secure approach. What I like about `nginx` is that it is very easy to add ssl support and to make the public facing api very secure. I understand why they did not add it though, it would have creeped the code base. If you need the functionality make sure that you do it right. The problem is that I had to figure out how to do it right. It's not as easy as just use `nginx`. It's like reading a technical journal, "It is trivial to show..." yeah, not to me its trivial...

Reply:
Yep! Agreed. The troubles with open source development I suppose. Good reasons on either side, and in their defense we're obviously able to find solutions without requiring them to include extra code. Anyway, thanks for adding the extra details, nice to have the background for why there needs to be this complicated work around. I've had similar problems with custom nginx proxies in the past, so I feel your pain!